.gitlab-ci.yml

```diff
@@ -7,10 +7,6 @@ stages:
 default:
   retry:
     max: 2
-    when:
```

Comment:
Maybe it's worth trying to add `stuck_or_timeout_failure` here?

Reply:
Don't think so. Actually those jobs should never fail. So I believe `always` is the right choice